src/main/java/org/cyclops/integrateddynamics/core/evaluate/variable/ValueTypeCategoryNumber.java

```diff
@@ -147,6 +147,31 @@ public IValue min(IVariable a, IVariable b) throws EvaluationException {
         );
     }
 
+    public IValue increment(IVariable a) throws EvaluationException {
+        IValueTypeNumber type = getType(a);
+        return type.increment(castValue(type, a.getValue()));
+    }
+
+    public IValue decrement(IVariable a) throws EvaluationException {
+        IValueTypeNumber type = getType(a);
+        return type.decrement(castValue(type, a.getValue()));
+    }
+
+    public IValue modulus(IVariable a, IVariable b) throws EvaluationException {
+        IValueTypeNumber type = getLowestType(getType(a), getType(b));
+        IValue bValue = castValue(type, b.getValue());
+        if (type.isZero(bValue)) { // You can not divide by zero
+            throw new EvaluationException(new TranslatableComponent(L10NValues.OPERATOR_ERROR_DIVIDEBYZERO));
+        } else if (type.isZero(bValue)) { // If b is neutral element for division
+            return type.getDefault(); // ? maybe each type should have a 'zero' value defined, rather than just using the default value here
```

Comment:
The default value of all current number types is a zero value so I used that here, but I don't know if that's the strictly intended behaviour - should a `getZero` method be added to the number types and used here?

Reply:
Yeah, zero is safe to assume for the default value. Perhaps we may want to add it to the javadoc even.